source/server/connection_handler_impl.cc

```diff
@@ -38,7 +38,8 @@ void ConnectionHandlerImpl::addListener(absl::optional<uint64_t> overridden_list
       }
       NOT_REACHED_GCOVR_EXCL_LINE;
     }
-    auto tcp_listener = std::make_unique<ActiveTcpListener>(*this, config);
+    auto tcp_listener = std::make_unique<ActiveTcpListener>(
+        *this, config, worker_index_.has_value() ? *worker_index_ : 0);
```

Comment:
When does `worker_index_` have no value? Only when handling hot restart, right?

Reply:
For admin, I will add a comment.